src/state/CleanUpState.ts

```diff
@@ -54,6 +87,13 @@ export class CleanUpState implements IState{
         }
     }
 
+    /**
+     * Reverts the properties of the mirror node.
+     * 
+     * This method cleans up unneeded mirror node properties and writes the updated properties back to the file.
+     * @private
+     * @returns {void}
```

Comment:
I think when we have a void method we do not need a return. As you did it in src/services/ServiceLocator.ts